src/tests/ftest/util/server_utils.py

```diff
@@ -121,10 +121,11 @@ def update_pattern(self, mode, host_qty):
         """
         if mode == "format":
             self.pattern = self.FORMAT_PATTERN
-            self.pattern_count = host_qty
+        elif mode == "reformat":
+            self.pattern = self.REFORMAT_PATTERN
```

Comment:
Not sure if pylint will complain about this one.  The `self.REFORMAT_PATTERN` definition was added in https://github.com/daos-stack/daos/pull/2645, but it does not make sense to cherry-pick that as well.  If it is a problem I'd either remove these lines (preferred) or add [this line](https://github.com/daos-stack/daos/blob/dcb7924185c9cc41da80392f9dcda076dcfcbf41/src/tests/ftest/util/server_utils.py#L42).